examples/value_list.rs

```diff
@@ -20,14 +20,14 @@ async fn main() {
         .unwrap();
 
     #[derive(scylla::ValueList)]
-    struct MyType {
+    struct MyType<'a> {
```

Comment:
Does your PR make it possible to use type parameters as well (not only lifetime parameters)? If so, could you extend the example and add another structure which is generic over a type parameter?

Reply:
Yes it does, and given the way split_for_impl works if would be hard for someone to break one without breaking the other.

Reply:
23695bbc104959f183e12a6ab3f74f75ad8baefd